api/db-cdr/generate-cdr/run-achilles-queries.sh

```diff
@@ -573,6 +583,7 @@ COUNT(distinct p1.PERSON_ID) as source_count_value from \`${BQ_PROJECT}.${BQ_DAT
 on p1.person_id = co1.person_id
 where (extract(year from current_date()) - p1.year_of_birth) >= 18 and
 (extract(year from current_date()) - p1.year_of_birth) < 30
+and co1.procedure_source_concept_id not in (select distinct procedure_concept_id from \`${BQ_PROJECT}.${BQ_DATASET}.procedure_occurrence\`)
```

Comment:
Some of these look slightly different than the others as it didn't already have `procedure_concept_id != prcedure_source_concept_id` - please double check